<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestQString</class>
 <widget class="QMainWindow" name="TestQString">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>结果</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edResult">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="cbContains">
        <property name="text">
         <string>contains()</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labCount">
          <property name="text">
           <string>count()</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBCount"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Str1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbStr1">
          <property name="minimumSize">
           <size>
            <width>550</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>Str1</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Str2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbStr2">
          <property name="minimumSize">
           <size>
            <width>550</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>str222222</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>字符串</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QPushButton" name="pBtnToup">
             <property name="text">
              <string>toUpper</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pBtnAppend">
             <property name="text">
              <string>Append</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="pBtnLeft">
             <property name="text">
              <string>left</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pBtnPrepend">
             <property name="text">
              <string>Prepend</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pBtnToLow">
             <property name="text">
              <string>toLower</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="pBtnSection">
             <property name="text">
              <string>section</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="pBtnRight">
             <property name="text">
              <string>right</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="pBtnTrimmed">
             <property name="text">
              <string>trimmed</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="pBtnSplite">
             <property name="text">
              <string>splite</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="pBtnSim">
             <property name="text">
              <string>simplified</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>数字</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>27</y>
           <width>99</width>
           <height>170</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="bBtnCount">
            <property name="text">
             <string>count</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtnSize">
            <property name="text">
             <string>size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtnCap">
            <property name="text">
             <string>capacity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtnIndex">
            <property name="text">
             <string>indexOf</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBtnLastIndex">
            <property name="text">
             <string>lastIndexOf</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>逻辑判断</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>27</y>
           <width>195</width>
           <height>100</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="pBtnEnds">
            <property name="text">
             <string>endsWith</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pBtnStart">
            <property name="text">
             <string>startWith</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pBtnContains">
            <property name="text">
             <string>contains</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pBtnNull">
            <property name="text">
             <string>isNull</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pBtnEmpty">
            <property name="text">
             <string>isEmpty</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
